src/scss/partials/_helpers.scss

```diff
@@ -12,48 +12,22 @@
   flex-wrap: nowrap;
 }
 
-@media screen and (min-width: 576px) {
-  .container--limited-sm {
-    max-width: 540px;
-  }
-}
-
-@media screen and (min-width: 768px) {
-  .container--limited-md {
-    max-width: 720px;
-  }
-}
-
-@media screen and (min-width: 992px) {
-  .container--limited-lg {
-    max-width: 960px;
-  }
-}
-
-@media screen and (min-width: 1200px) {
-  .container--limited-xl {
-    max-width: 1140px;
+@each $breakpoint, $container-max-width in $container-max-widths {
+  @include media-breakpoint-up($breakpoint, $grid-breakpoints) {
+    .container--limited-#{$breakpoint} {
+      max-width: $container-max-width;
+    }
+
+    .w-#{$breakpoint}-auto {
+      width: auto;
+    }
+
+    .w-#{$breakpoint}-100 {
+      width: 100%;
+    }
```

Comment:
I bet this is something that would probably interest the bootstrap team  isn't it?

Reply:
[They already use it](https://github.com/twbs/bootstrap/blob/main/scss/_containers.scss) :)
And that's what I've been trying to say from the start of this theme. Bootstrap has a lot already done just waiting to be used. No need to create duplications, when almost whole theme could be done just by setting up default variables. Not much places where you actually need some custom css.

Reply:
```scss
@each $name, $gray-variant in $grays {
  .text-gray-#{$name} {
    color: $gray-variant;
  }
}
```
This would generate `.text-gray-100`, `.text-gray-200`, etc. just like Tailwind. Would this also be useful or is `.text-light`, `text-dark` (both default BS) and `.text-gray` (theme helpers) enough?

Reply:
That's why I'm only asking if this would be helpful right now. I can imagine few places where you would want text with different shades of gray, but that's just a suggestion if it's ever needed.